### PR TITLE
Upgrade to core 1.1.1

### PR DESCRIPTION
**Pull request by tgoyne**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -25,6 +26,9 @@ x.x.x Release notes (yyyy-MM-dd)
 * Realm now throws an exception if an `Object` subclass is defined with a managed Swift `lazy` property.
   Objects with ignored `lazy` properties should now work correctly.
 * Update the LLDB script to work with recent changes to the implementation of `RLMResults`.
+* Fix an assertion failure when a Realm file is deleted while it is still open,
```
Comment (jpsim) [inline CHANGELOG.md]: seems odd to bring attention to a bug fix when doing something unsupported

**Comment by jpsim** [comment]
Doesn't the fix "Fixed a recently introduced crash bug on indexed columns (#1869)" also apply to us?

**Comment by tgoyne** [comment]
We never shipped a release with the broken core version.

**Comment by jpsim** [comment]
👍

**Pull request merged**